presto-main/src/main/java/com/facebook/presto/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -252,14 +250,12 @@ protected Node visitNode(Node node, Void context)
             return node;
         }
 
-        private Constraint<ColumnHandle> getConstraint(QuerySpecification specification)
+        private Constraint<ColumnHandle> getConstraint(QuerySpecification specification, Plan plan)
         {
             if (!specification.getWhere().isPresent()) {
```

Comment:
Is this still needed? Remove this check, so you can remove the `specification` parameter.